.github/workflows/lint.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint
```

Comment:
I would mention in the name the fact that this is not just lint checks, but also auto-fixes.